frontend/src/main/web/src/components/Row.js

```diff
@@ -25,7 +25,10 @@ const classes = {
     }
   }
 }
-
+/**
+ * A flexbox component that align all elements vertically in a row.
```

Comment:
- [ ] "vertically" indicates that everything will be stacked up on top of each other, is that right?
- [ ] is "elements" talking about the elements inside it (children), or talking about if I have multiple `<Row>` elements next to each other in a container?